ethcore/light/src/client/header_chain.rs

```diff
@@ -788,7 +789,7 @@ mod tests {
 			header.set_parent_hash(parent_hash);
 			header.set_number(i);
 			header.set_timestamp(rolling_timestamp);
-			header.set_difficulty(*genesis_header.difficulty() * i.into());
+			header.set_difficulty(*genesis_header.difficulty() * i as u32);
```

Comment:
perhaps change the loop range to be `u32` instead?

Reply:
there is also `header.set_number(i)` which uses `i`. So one of them have to be cased